src/main/kotlin/com/kotlin/boardproject/global/service/CustomUserDetailsService.kt

```diff
@@ -1,11 +1,11 @@
-package com.kotlin.boardproject.service
+package com.kotlin.boardproject.global.service
 
-import com.kotlin.boardproject.auth.UserPrincipal
-import com.kotlin.boardproject.repository.UserRepository
+import com.kotlin.boardproject.domain.user.repository.UserRepository
```

Comment:
🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end